src/main/resources/templates/start.html

```diff
@@ -7,14 +7,14 @@
     <meta charset="UTF-8">
     <link rel="icon" href="https://www.shareicon.net/data/2017/01/18/872963_game_512x512.png">
     <title>Hobby Collection</title>
+    <link th:href="@{/css/style.css}" type="text/css" rel="stylesheet">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
 </head>
 <body>
     <div th:replace="fragments/reusable-components :: navbar"></div>
-    <p style="padding-bottom:1%"></p>
 
-    <div class="container">
+    <div class="container main-div">
```

Comment:
In-line styling can create technical debt, as we will need to go and upgrade each invidual tag in order to make changes the the CSS of the page. Instead, we should use CSS files for our CSS code. This will save us time and energy as we will not have to search for where those individual tags.

The in-line styling was also creating issues as there were empty <p> tags that were merely being used to create spacing from the divs. Either padding needs to be used on the divs themselves, or we should be looking to use the <br> tag as a last resort. It is usually considered best practice to rely on CSS over HTML to control the presentation of the page.

Citations: [https://www.lifewire.com/avoid-inline-styles-for-css-3466846](https://www.lifewire.com/avoid-inline-styles-for-css-3466846), [https://stackoverflow.com/questions/2931082/is-it-always-bad-idea-to-use-inline-css-for-used-once-property](https://stackoverflow.com/questions/2931082/is-it-always-bad-idea-to-use-inline-css-for-used-once-property)

Reply:
Good idea.  Use linked CSS instead of inline/embedded.  More reusable.